src/lib/import_export/csv/csv_converter.hpp

```diff
@@ -59,6 +59,11 @@ class CsvConverter : public BaseCsvConverter {
       Assert(_config.null_handling != NullHandling::RejectNullStrings,
              "Unquoted null found in CSV file. Quote it for string literal \"null\", leave field empty for null "
              "value, or set 'null_handling' to the appropriate strategy in parse config.");
+      if (_config.null_handling == NullHandling::NullStringAsNull) {
+        Assert(_is_nullable,
+               "Unquoted null found in CSV file, while the associated column is not nullable. Quote it for string "
+               "literal \"null\".");
```

Comment:
The suggestion could work. But if NULL occurs in a non-nullable column the user might also consider making the column nullable?

Reply:
Hm, that's true. How about: ` Quote it for string literal \"null\" or make the column nullable.`?

Reply:
Works for me.

Reply:
I would still keep the first sentence